src/inc/check.h

```diff
@@ -419,6 +416,9 @@ CHECK CheckValue(TYPENAME &val)
 
 #endif
 
+#define CONSISTENCY_CHECK(_condition) \
```

Comment:
In error, when I got the error (which I later resolved with the additional parenthesis), it led me to believe that CONSISTENCY_CHECK_MSG was undefined, was trying to ensure that Clang wasn't evaluating the macro strangely. I'll revert this change.